nilearn/glm/contrasts.py

```diff
@@ -40,7 +40,15 @@ def expression_to_contrast_vector(expression, design_columns):
         contrast_vector[list(design_columns).index(expression)] = 1.
         return contrast_vector
     df = pd.DataFrame(np.eye(len(design_columns)), columns=design_columns)
-    contrast_vector = df.eval(expression, engine="python").values
+    try:
+        contrast_vector = df.eval(expression, engine="python").values
+    except AttributeError:
```

Comment:
depending on the column names and exact contrast the error might be something else -- for example if a column name is a keyword it will be a SyntaxError. you can add SyntaxError to the list or maybe catch a broader type of exception